src/Response/Response.php

```diff
@@ -0,0 +1,46 @@
+<?php declare(strict_types=1);
+
+namespace DoclerLabs\ApiClientBase\Response;
+
+class Response
+{
+    /** @var int */
+    private $statusCode;
+
+    /** @var array */
+    private $payload;
+
+    /** @var array */
+    private $headers;
+
+    public function __construct(int $statusCode, array $payload = [], array $headers = [])
+    {
+        $this->statusCode = $statusCode;
+        $this->payload    = $payload;
+        $this->headers    = $headers;
+    }
+
+    /**
+     * @return array
```

Comment:
not an array

- [ ] int

Reply:
Actually I think the doc comments could be dropped, they just duplicate type hints and not adding anything new.

Reply:
Totally, doc comments are removed